### DISCUSSION TITLE
「开源之书·共读」#41 《无需法律的秩序:相邻者如何解决纠纷？》 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#44] [Announcements]
# 「开源之书·共读」#41 《无需法律的秩序:相邻者如何解决纠纷？》 - 「开源之道」

https://opensourceway.community/posts/book-of-open-source/41th-communities-in-cyberspace/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

又一个线头儿...
一切只能重新开始,
以往在中国环境中积累的经验, 只能先放下,
才能更好的回来理解, 现实行为背后的精神...
